mne/io/base.py

```diff
@@ -1923,6 +1923,30 @@ def describe(self, data_frame=False):
             msg += f"{cols['max'][i]:>8.2f}"
             print(msg)
 
+    @verbose
+    def rename_annotations(self, mapping, verbose=None):
+        """Rename annotation description(s). Operates inplace.
+
+        Parameters
+        ----------
+        mapping : dict
+            A dictionary mapping the old description to a new description
+            name e.g. {'1.0' : 'Control', '2.0' : 'Stimulus'}.
+        %(verbose_meth)s
+
+        Returns
+        -------
+        self : mne.Annotations
+            The modified Annotations object.
+
+        Notes
+        -----
+        .. versionadded:: 0.24.0
+        """
+        self.annotations.description = _rename_annotations(
+            self.annotations.description, mapping, verbose)
+        return self
```

Comment:
is there a particular reason not to use the public function here?  E.g.:
```suggestion
        self.annotations.rename(mapping, verbose=verbose)
        return self
```
?  Seems slightly simpler to me / makes it clearer that it's just an alias.